packages/docusaurus-theme-classic/src/theme/NavbarItem/DocSidebarNavbarItem.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import DefaultNavbarItem from '@theme/NavbarItem/DefaultNavbarItem';
+import {useLatestVersion, useActiveDocContext} from '@theme/hooks/useDocs';
+import clsx from 'clsx';
+import {getInfimaActiveClassName} from './index';
+import type {Props} from '@theme/NavbarItem/DocSidebarNavbarItem';
+import {useDocsPreferredVersion, uniq} from '@docusaurus/theme-common';
+import type {GlobalDataVersion} from '@docusaurus/plugin-content-docs';
+
+function getSidebarInVersion(versions: GlobalDataVersion[], sidebarId: string) {
+  const allSidebars = versions.map((version) => version.sidebars)[0];
+  if (!allSidebars) {
+    throw new Error(`DocSidebarNavbarItem: couldn't find any sidebar`);
```

Comment:
this error can't happen, map/flatMap always returns an array. At best the array would be empty, but it's probably not so useful anyway